docs/reference/export.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: default
+title: Exporting data
+description: Use lakeFS spark client to export lakeFS commit to the object store. 
+parent: Reference
+nav_exclude: true
+has_children: false
+---
+
+# Exporting data
+{: .no_toc }
+The export operation copies all data from a given lakeFS commit to 
+a designated object store location.
+
+For instance, the contents `lakefs://example@master` might be stored on
+`s3://company-bucket/example/latest`.  Clients entirely unaware of lakeFS could use that
+base URL to access latest files on `master`.  Clients aware of lakeFS can continue to use
+the lakeFS S3 endpoint to access repository files on `s3://example/master`, as well as
+other versions and uncommitted versions.
+
+Possible use-cases:
+1. External consumers of data don't have access to your lakeFS installation.
+1. Some data pipelines in the organization are not fully migrated to lakeFS.
+1. You want to experiment with lakeFS as a side-by-side installation first.
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+## How to use
+
+Set up lakeFS spark metadata client with the endpoint and credentials as instructed in the previous [page](./spark-client.md).
+
+The client exposes the `Exporter` object with 3 export options:
+
+1. Exporting *all* objects at the HEAD of a given branch. Does not include
```

Comment:
I think this should go in the imperative mode.  @peacing as a native speaker WDYT?

```suggestion
1. Export *all* objects at the HEAD of a given branch. Does not include
```

Reply:
Done.